rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -22,15 +24,40 @@ setup:
         refresh: true
         body:
          - {"index": {}}
-         - { "double" : 42.1, "long": 25 }
+         - { "double" : 42.1, "long": 25, "float": 0.01 }
          - {"index": {}}
-         - { "double" : 100.7, "long": 80 }
+         - { "double" : 100.7, "long": 80, "float": 0.03 }
          - {"index": {}}
-         - { "double" : 50.5, "long":  75}
+         - { "double" : 50.5, "long":  75, "float": 0.04 }
 # For testing missing values
          - {"index": {}}
          - {}
 
+---
+# Regression test for 77033
```

Comment:
When I see this comment in 9 months I'm super unlikely to look up the issue- I'll either `git-blame` the code or puzzle it out from the title of the test or body of it. I don't think the link adds anything here.

Reply:
yeah, okay.  Do you also think it's not worth having a similar comment in the unit tests I added?

Reply:
I don't. Honestly if you like the comments you can keep 'em. But I tend to skip those sorts of comments when reading.